### PR TITLE
:memo: 40 Add working exemple on kubernetes

### PR DESCRIPTION
**Pull request by mathieuHa**

### CODE REVIEW

File: exemples/kubernetes/crowdsec/values.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+image.tag: ""
```
Comment (maxlerebourg) [inline exemples/kubernetes/crowdsec/values.yaml:1]: end of file

**Comment by mathieuHa** [comment]
```time="2022-12-30T18:00:47Z" level=error msg="Plugins are disabled because an error has occurred." error="mkdir plugins-storage: read-only file system"```

Error of plugins storage disabled

**Comment by mathieuHa** [comment]
plugins must be enabled in values.yml
```
experimental:
  plugins:
    enabled: true
```

**Comment by mathieuHa** [comment]
Will help #67

File: exemples/kubernetes/traefik/plugin.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+    name: bouncer
+    namespace: traefik
+spec:
+    plugin:
+        bouncer:
+            CrowdsecLapiKey: 40796d93c2958f9e58345514e67740e5
+            Enabled: "true"
```
Comment (maxlerebourg) [inline exemples/kubernetes/traefik/plugin.yml:10]: why this is a string

Comment (mathieuHa) [reply exemples/kubernetes/traefik/plugin.yml:10]: You mean why is it hardcoded ?

File: exemples/kubernetes/whoami/whoami-ingress.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: whoami
+  namespace: "traefik"
```
Comment (maxlerebourg) [inline exemples/kubernetes/whoami/whoami-ingress.yml:5]: ??

File: exemples/kubernetes/whoami/whoami-ingress.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  - http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: whoami
+            port:
+              name: web
+
```
Comment (maxlerebourg) [inline exemples/kubernetes/whoami/whoami-ingress.yml:21]: no eof

File: exemples/kubernetes/whoami/whoami-services.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  namespace: traefik
+
+spec:
+  ports:
+    - name: web
+      port: 80
+      targetPort: web
+
+  selector:
+    app: whoami
```
Comment (maxlerebourg) [inline exemples/kubernetes/whoami/whoami-services.yml:14]: no eof

**Pull request merged**